README.md

```diff
@@ -1,9 +1,95 @@
-# Weather plugin for CodeX Bot platform
+## How to create a new application for CodeX Bot
 
-Allow you to be informed about weather conditions in your region via Telegram chat
+This article is about new CodeX Bot application development. Throughout this guide we build an application that can provide weather forecast for different periods of time.
 
-## CodeX Team
+For simplicity we use our modern SDK for Python (**link**). We also provide instructions how to run your own production-ready CodeX Bot server.
 
-https://ifmo.su
+Before start you need to run CodeX Bot Core (read [Developer's Guide](https://github.com/codex-team/codex.bot/wiki/Developer's-Guide)) or use our [@codex_bot](https://t.me/codex_bot)
```

Comment:
Before you start, we need to run @codex_bot Core (read [Developer's Guide](https://github.com/codex-team/codex.bot/wiki/Developer's-Guide)) or use our [@codex_bot](https://t.me/codex_bot)